src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -4039,4 +4039,20 @@ inline RppStatus compute_packed_to_planar_host(T* srcPtr, RppiSize srcSize, T* d
     return RPP_SUCCESS;
 }
 
+inline RppStatus compute_brightness_24_host(__m256 &pVecR, __m256 &pVecG, __m256 &pVecB, __m256 *pBrightnessParams)
```

Comment:
This file has a lot of BatchPD and Tensor functions. Could you add the new tensor functions needed right under L2069? Latest one first - so right above the compute_gridmask_masks_16_host(). We'll fix this during a restructure.